test/models.js

```diff
@@ -77,7 +77,12 @@ describe('models', function () {
     };
     var model = new Model('Sample', definition);
 
-    expect(model.getMockSignature()).toEqual('<span class="strong">Sample {</span><div><span class="propName false">id</span> (<span class="propType">integer</span>, <span class="propOptKey">optional</span>),</div><div><span class="propName false">name</span> (<span class="propType">string</span>, <span class="propOptKey">optional</span>),</div><div><span class="propName false">photos</span> (<span class="propType">Array[object]</span>, <span class="propOptKey">optional</span>)</div><span class="strong">}</span>');
```

Comment:
Testing the generated HTML is a pain in the butt but I don't think removing them makes much sense here.  The tests you've replaced them with are quite lax and do not ensure the structure created is what is required.  While I do think we need some utilities for easing the pain of this, I'm not sure making things more lax makes sense.

This comment applies to all similar changes below so I will not be marking the other places.